resources/js/actions/userAuthActions.js

```diff
@@ -0,0 +1,77 @@
+import userApi from '../apis/userApi';
+import { SIGN_IN } from './types';
+
+export const signIn = (formData) => async dispatch => {
+    var data = {};
+    var submitDisabled = true;
+    const email = formData.get('email');
+    const password = formData.get('password');
+    const users = {
+        email: email,
+        password: password
+    }
+    // if all fields are not blank 
+    if (users.email != '' && users.password != '') {
```

Comment:
use strict equality/inequality operators

```suggestion
    if (users.email !== '' && users.password !== '') {
```

Reply:
Made changes to userAuthActions.js. This is resolved.